src/cloud-functions.ts

```diff
@@ -179,7 +235,7 @@ export interface Resource {
   labels?: { [tag: string]: string };
 }
 
-/**
+/** @hidden
```

Comment:
This should always be on the second line with first line being `/**`, unless it's a one-line comment like `/** @hidden */`

Reply:
SG.  With no empty lines before nor after?